### DISCUSSION TITLE
Downloaded latest .Net Framework but OnlyM won't open?

### DISCUSSION THREAD

**Discussion by captaintech** [AntonyCorbett/OnlyM#416] [Q&A] [answered]
Hello everyone,

Upon opening OnlyM lately it asks to download the 5.0.0 framework. I have the downloaded it and then go to open OnlyM again but it just asks to download it again. Would anyone happen to have a fix for this? I am on Windows 10. Thanks everyone!

**Comment by AntonyCorbett** [comment, marked as answer]
You need the .net 5 x86 desktop runtime. The latest OnlyM installer should download it automatically.

**Comment by captaintech** [comment]
Thank you so much brother! Not sure what I was doing wrong the other night
but this fixed it! Much appreciated!
